src/Schema/aspire-8.0.json

```diff
@@ -73,6 +73,54 @@
               "bindMounts": {
                 "$ref": "#/definitions/bindMounts"
               },
+              "volumes": {
```

Comment:
Should we update this file name? `aspire-8.0.json` seems like what was supported in Aspire 8.0. But now that we have new content in here, revving the version seems appropriate.